batch7crud-roll7/src/main/java/com/lftechnology/batch7crud/filter/UserFilter.java

```diff
@@ -0,0 +1,36 @@
+package com.lftechnology.batch7crud.filter;
+
+import com.lftechnology.batch7crud.controller.CommonConstants;
+import javax.servlet.*;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
+import java.io.IOException;
+
+/**
+ * Created by leapfrog on 1/27/16.
+ */
+public class UserFilter implements Filter {
+  private String LOGIN_ACTION_URI;
+
+  @Override
+  public void init(FilterConfig fConfig) throws ServletException {
+    LOGIN_ACTION_URI = fConfig.getInitParameter("loginActionURI");
+  }
+
+  @Override
+  public void destroy() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1186)